lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -375,21 +411,43 @@ static void inlineInputOnly(hw::HWModuleOp oldMod,
     hw::InstanceGraphNode *instParentNode = instanceGraph.lookup(instParent);
     SmallVector<Operation *, 16> lateBoundOps;
     b.setInsertionPoint(inst);
+    // Namespace that tracks inner symbols in the parent module.
+    hw::ModuleNamespace nameSpace(instParent);
+    // A map from old inner symbols to new ones.
+    DenseMap<mlir::StringAttr, mlir::StringAttr> symMapping;
+
     for (auto &op : *oldMod.getBodyBlock()) {
       // If the op was erased by instance extraction, don't copy it over.
       if (opsToErase.contains(&op))
         continue;
 
+      // If the op has an inner sym, first create a new inner sym for it.
+      if (auto innerSym = op.getAttrOfType<StringAttr>("inner_sym")) {
+        auto newName = b.getStringAttr(nameSpace.newName(innerSym.getValue()));
+        auto result = symMapping.insert({innerSym, newName});
+        assert(result.second);
```

Comment:
Yeah, was wondering if the rename-multiple-times bugs/concerns we've encountered in ModuleInliner would apply here.  Is that possible, do you know?

Reply:
I think this is fine unless the original module doesn't contain non-unique inner symbols. Though there is no verification for IR so probably we need better messages.